### ISSUE TITLE
Build of module server in branch release/v6.4.0 fails with link error on Ubuntu 20.04

### ISSUE THREAD

**Issue by larrychen8276** [ONLYOFFICE/build_tools#376]
The build of module server in branch release/v6.4.0 fails with link error on Ubuntu 20.04. The sequenve leading to this is

git clone <build_tools>
git checkout release/v6.4.0.121 in ./build_tools
cd tools/linux in ./build_tools
./automate.py server
or
python3 ./automate.py

```
Done. Made 92 targets from 56 files in 118ms
ninja: Entering directory `out.gn/linux_64'
ninja: no work to do.
Traceback (most recent call last):
  File "fetch.py", line 5, in <module>
    import config
ImportError: No module named config
[git] update: DocumentBuilder
branch does not exist...
switching to master...
[git] update: core
branch does not exist...
switching to master...
[git] update: plugin-thesaurus
branch does not exist...
switching to master...
[git] update: web-apps
branch does not exist...
switching to master...
[git] update: document-server-integration
branch does not exist...
switching to master...
[git] update: plugin-translator
branch does not exist...
switching to master...
[git] update: dictionaries
branch does not exist...
switching to master...
[git] update: sdkjs
branch does not exist...
switching to master...
[git] update: plugin-highlightcode
branch does not exist...
switching to master...
[git] update: core-fonts
branch does not exist...
switching to master...
[git] update: server
branch does not exist...
switching to master...
[git] update: desktop-sdk
branch does not exist...
switching to master...
[git] update: plugin-photoeditor
branch does not exist...
switching to master...
[git] update: plugin-mendeley
branch does not exist...
switching to master...
[git] update: desktop-apps
branch does not exist...
switching to master...
[git] update: document-templates
branch does not exist...
switching to master...
[git] update: plugin-macros
branch does not exist...
switching to master...
[git] update: plugin-zotero
branch does not exist...
switching to master...
[git] update: plugin-youtube
branch does not exist...
switching to master...
[git] update: plugin-ocr
branch does not exist...
switching to master...
[git] update: plugin-speech
branch does not exist...
switching to master...
[fetch & build]: boost
[fetch & build]: cef
[fetch & build]: icu
[fetch & build]: openssl
[fetch & build]: v8
gcc major version: 9, use clang:true
gn gen out.gn/linux_64 --args="is_debug=false target_cpu=\"x64\" v8_target_cpu=\"x64\" v8_static_library=true is_component_build=false v8_use_snapshot=false is_clang=true use_sysroot=false treat_warnings_as_errors=false"
Error (python): 1
---------------------------------------------
build branch: 
---------------------------------------------
---------------------------------------------
build modules: desktop builder server
---------------------------------------------
Error (./make.py): 1
```

**Comment by ShockwaveNN** [comment]
We released v6.4.0 and removed this branch from all repos

So you should build from master

**Comment by larrychen8276** [comment]
> We released v6.4.0 and removed this branch from all repos
> 
> So you should build from master

build master have been solve the error

**Comment by ShockwaveNN** [comment]
In that case I'll close that issue
